components/Chord/ChordDiagram.vue

```diff
@@ -0,0 +1,300 @@
+<template>
+  <svg
+    :id="chartDomID + '-svg'"
+    :width="meta.width"
+    :height="meta.height"
+    :fill-opacity="meta.fillOpacity"
+    class="svg chord"
+  >
+    <!--SUNBURST-->
+    <transition-group
+      id="sunburstSlices"
+      tag="g"
+      name="fade"
+      :duration="transitionDuration"
+      :transform="'translate(' + radius + ',' + radius + ')'"
+    >
+      <!--SLICES-->
+      <g v-for="(arc, index) in root" :key="index">
+        <path
+          class="line"
+          :fill="sliceColor(arc)"
+          :fill-opacity="line.fillOpacity"
+          :stroke="line.stroke"
+          :stroke-width="line.stroke_width"
+          :d="arcFunction(arc)"
+        >
+          <title>
+            {{ ancestorPath(arc) }}
+          </title>
+        </path>
+        <text
+          pointer-events="null"
+          text-anchor="middle"
+          font-family="sans-serif"
+          font-size="8"
+          style="user-select: none;"
+          dy="0.35em"
+          class="sunburst-text"
+          textLength="10"
+          :textLength="textLength(arc)"
+          lengthAdjust="spacingAndGlyphs"
+          :transform="labelTransform(arc)"
+        >
+          {{ arc.data.name }}
+        </text>
+      </g>
+    </transition-group>
+    <!--
+    &lt;!&ndash;TEXT&ndash;&gt;
+    <transition-group
+      id="SunburstLables"
+      tag="g"
+      name="fade"
+      :duration="transitionDuration"
+      :transform="'translate(' + radius + ',' + radius + ')'"
+    >
+      <g v-for="(item, index) in root" :key="index">
+        <text
+          pointer-events="null"
+          text-anchor="middle"
+          font-family="sans-serif"
+          font-size="8"
+          style="user-select: none;"
+          dy="0.35em"
+          class="sunburst-text"
+          textLength="40"
+          lengthAdjust="spacingAndGlyphs"
+          :transform="labelTransform(item)"
+        >
+          {{ item.data.name }}
+        </text>
+      </g>
+    </transition-group>
+  -->
+    <!--Bubbles-->
+    <transition-group
+      id="Bubbles"
+      tag="g"
+      name="fade"
+      :duration="transitionDuration"
+    >
+      <g v-for="(item, index) in users" :key="index">
+        <circle
+          :cx="radius"
+          :cy="radius"
```

Comment:
@parinaznasr Why are you only giving `radius` to the circle’s `cx` and `cy`? It should be something like this:
```
:cx=“item.someThingThatHasACoordinateRelativeToX”
:cy=“item.someThingThatHasACoordinateRelativeToY”

```

Reply:
It's not finished yet, I'm still working on this part :)